### ISSUE TITLE
add_test_code

### ISSUE THREAD

**Issue by amy-hyunji** [korean-fingerspelling-recognition/hand_gesture_recognition#3]
## expr1: batch_size 128 / image_size 48
flags.DEFINE_boolean("append_handmask", True, 'If true, train with handmask appended')
flags.DEFINE_integer("image_size", 48, "define input image size")
flags.DEFINE_float("dropout", 0.5, "dropout rate")
flags.DEFINE_integer("display_step", 1, "define display_step")
flags.DEFINE_boolean("use_bottleneck", False, "change the sub-layer on Resnet")
flags.DEFINE_string("saved_path", "Default", "path to save model: Will create automatically if it does not exist")
flags.DEFINE_integer("num_steps", 555555500, "define the number of steps to train")
flags.DEFINE_boolean("train_mode", True, "True if in train mode, False if in Test mode")
flags.DEFINE_string("data_path", "./result", "path of the dataset: inside this directory there should be files named with label")
flags.DEFINE_float("learning_rate", 0.0001, "learning_rate during training")
flags.DEFINE_integer("batch_size", 128, "batch size during training")
flags.DEFINE_integer("save_steps", 500, "number of save steps")
flags.DEFINE_integer("dim", 4, "input_channel is 4")
flags.DEFINE_integer("num_classes", 8, "number of classes")
flags.DEFINE_integer("model_num", 1, "0: default model / 1: resnet / 2: bottlenect resnet")

## expr2: batch_size 64 / image_size 64
flags.DEFINE_boolean("append_handmask", True, 'If true, train with handmask appended')
flags.DEFINE_integer("image_size", 64, "define input image size")
flags.DEFINE_float("dropout", 0.5, "dropout rate")
flags.DEFINE_integer("display_step", 1, "define display_step")
flags.DEFINE_boolean("use_bottleneck", False, "change the sub-layer on Resnet")
flags.DEFINE_string("saved_path", "Default", "path to save model: Will create automatically if it does not exist")
flags.DEFINE_integer("num_steps", 555555500, "define the number of steps to train")
flags.DEFINE_boolean("train_mode", True, "True if in train mode, False if in Test mode")
flags.DEFINE_string("data_path", "./result", "path of the dataset: inside this directory there should be files named with label")
flags.DEFINE_float("learning_rate", 0.0001, "learning_rate during training")
flags.DEFINE_integer("batch_size", 64, "batch size during training")
flags.DEFINE_integer("save_steps", 500, "number of save steps")
flags.DEFINE_integer("dim", 4, "input_channel is 4")
flags.DEFINE_integer("num_classes", 8, "number of classes")
flags.DEFINE_integer("model_num", 1, "0: default model / 1: resnet / 2: bottlenect resnet")

**Comment by amy-hyunji** [comment]
![overall graph](https://user-images.githubusercontent.com/44370759/58888001-a2696f00-8721-11e9-8704-d1e4c3e18c85.png)
![specific graph](https://user-images.githubusercontent.com/44370759/58888004-a2696f00-8721-11e9-878f-8c142859d079.png)
![tag](https://user-images.githubusercontent.com/44370759/58888116-d93f8500-8721-11e9-8584-832deea636d5.png)
